### PR TITLE
Add new fixtures for APUP + MTB

### PR DESCRIPTION
**Pull request by jimjohnsonrollings**
Resolves N/A

**Overall change:**
Creates more fixtures

**Code changes:**

- Adds fixtures for Arabic, Persian, Urdu, Pashto, Mundo, Brasil and Turkce

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by ryanmccombe — APPROVED** [review]

**Review by ghost — APPROVED** [review]

File: data/arabic/articles/c1er5mjnznzo.json [hunk 1]
```diff
@@ -0,0 +1,433 @@
...
+                      }
+                    }
+                  ]
+                }
+              }
+            ]
+          }
+        },
+        {
+          "type": "video",
```
Comment (chris-hinds) [inline data/arabic/articles/c1er5mjnznzo.json:190]: This block is missing the copyright holder which is required

```
Missing required property 'copyrightHolder' for ':article:content:model:blocks:video:model:blocks:aresMedia:model:blocks:image:model:blocks:rawImage:model'
```

**Review by chris-hinds — CHANGES_REQUESTED** [review]

File: data/arabic/articles/c1er5mjnznzo.json (hunk 1 above)
Comment (jimjohnsonrollings) [reply data/arabic/articles/c1er5mjnznzo.json:190]: 👍

**Review by chris-hinds — APPROVED** [review]

**Comment by chris-hinds** [comment]
As long a CI passes now that the fixture data is updated this is good to go

**Comment by hindsc52** [comment]
really not sure just increasing the timeout is the problem here. Are we receiving any other errors or messages in the build logs when we see this failure?

**Review by 12 — APPROVED** [review]

**Pull request merged**